Applications/WebPortal/WebApp/library/source/pages/account_page/profile_page/profile_model.ts

```diff
@@ -13,6 +13,12 @@ export abstract class ProfileModel {
   /** Returns the account's identity. */
   public abstract get identity(): Nexus.AccountIdentity;
 
+  /** Returns the account's readonly condition. */
+  public abstract get isReadonly(): boolean;
+
+  /** Returns the account's isPasswordFixed condition. */

```

Comment:
This doesn't help explain anything. What is a fixed password? What is a broken password?

Reply:
It's meant to indicate if the password can be changed. I'll rename it to isPasswordReadOnly.